include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -192,21 +194,26 @@ struct TypeInfo : Base<OrtTypeInfo> {
 };
 
 struct Value : Base<OrtValue> {
-  static Value CreateTensor(const OrtAllocatorInfo* info, void* p_data, size_t p_data_len, const int64_t* shape, size_t shape_len,
+  static Value CreateTensor(const AllocatorInfo& info, void* p_data, size_t p_data_len, const int64_t* shape, size_t shape_len,
                             ONNXTensorElementDataType type);
+  static Value CreateMap(Value& keys, Value& values);
```

Comment:
Should keys and values be passed by const-ref?

Reply:
The problem is that OrtCreateValue takes them as `OrtValue ** const` so I get a bunch of warnings if I do it that way.

Reply:
OK